internal/security/proxy/certs.go

```diff
@@ -58,9 +59,15 @@ func (cs certificate) Load() (*CertPair, error) {
 	if err != nil {
 		return nil, err
 	}
-	cp, err := cs.retrieve(t)
-	if err != nil {
-		return nil, err
+	var cp *CertPair
+	for {
```

Comment:
I don't typically like infinite loops that don't error out at some point. 
That being said, I think the ideal thing we want is for security-proxy-setup to use go-mod-secrets right? If that's the case this is probably fine for Fuji, but we shouldn't pull in this fix for master.